### ISSUE TITLE
Missing semicolon reported after two divison operators

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google/closure-linter#56]
```
$ cat semicolon-after-double-devision.js
var x = 2 / 2 /
        1;
$ gjslint semicolon-after-double-devision.js 
----- FILE  :  /home/vagrant/work/bidder/semicolon-after-double-devision.js 
-----
Line 1, E:0010: Missing semicolon at end of line
Found 1 errors, including 1 new errors, in 1 files (0 files OK).

Some of the errors reported by GJsLint may be auto-fixable using the script
fixjsstyle. Please double check any changes it makes and report any bugs. The
script can be run by executing:

fixjsstyle semicolon-after-double-devision.js 

This should not report an error and fixjsstyle breaks the code.

I tested it with gjslint 2.3.7 on Linux.
```

Original issue reported on code.google.com by `jonas.do...@gmail.com` on 9 Oct 2012 at 9:26
